WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -398,6 +401,7 @@ - (void)configureSections
                       @(PostSettingsSectionMeta),
                       @(PostSettingsSectionFormat),
                       @(PostSettingsSectionFeaturedImage),
+                      @(PostSettingsSectionStickyPost),
```

Comment:
I think we need to take care whether the site is self-hosted or not here. I seem to remember the WPKit part only took care of Jetpack/WPCom sites, so for self-hosted sites we'd show an option here that users can't actually change.

Reply:
@jklausa all *Self-hosted* blogs or only *Self-hosted non-Jetpack* blogs?
Let me know if this is correct:
In the first case I can check if the `post.blog` is `isHostedAtWPcom`.
In the second one I can check `post.blog.capabilities` exists. *Self-hosted non-Jetpack* blogs have no capabilities. This check is used by the `AztecPostViewController` when a new Editor Context is created for a given Post.

Reply:
Right, I think non-Jetpack blogs in this case!

Reply:
@jklausa done! 😄 Let me know it it's ok!